docs/CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## v2.0.0
+
+This release, most notably, adds support for TAP 15 - succinct hash bin delegation,
+which results in a few backwards-incompatible changes in the Metadata API.
```

Comment:
```suggestion
This release, most notably, adds support for [TAP 15] - succinct hash bin delegation,
which results in a few backwards-incompatible changes in the Metadata API.

**NOTE**: While TAP 15 has been accepted it is not yet part of the TUF specification.
Therefore, adopters should be prepared for potential changes to the implementation
in future and for a lack of support for TAP 15 in other TUF implementations.

[TAP 15]: https://github.com/theupdateframework/taps/blob/master/tap15.md
```

What do you think to adding something like this to try and ensure adopters are aware of the potential for changes in succinct hashed bins and to expect incompatibility or lack of support across implementations when using it?